<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>nom_utilisateur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mdp">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>650</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>mdp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_connecter">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>750</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>connecter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nom">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>570</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>-30</y>
     <width>941</width>
     <height>1011</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>le_mdp</zorder>
  <zorder>label_2</zorder>
  <zorder>pb_connecter</zorder>
  <zorder>le_nom</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
